gate-web/src/main/groovy/com/netflix/spinnaker/gate/security/saml/SamlSsoConfig.groovy

```diff
@@ -155,6 +156,12 @@ class SamlSsoConfig extends WebSecurityConfigurerAdapter {
         def email = assertion.getSubject().nameID.value
         def roles = extractRoles(email, attributes, userAttributeMapping)
 
+        if (samlSecurityConfigProperties.requiredRoles) {
+          if (!roles.containsAll(samlSecurityConfigProperties.requiredRoles)) {
```

Comment:
Was the previous check a containsAll? Not sure it particular matters to us right now but would seem to make more sense that we only require one of the roles.